webpack/assets/javascripts/react_app/components/HostDetails/Tabs/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import { translate as __ } from '../../../common/I18n';
 import { addGlobalFill } from '../../common/Fill/GlobalFill';
-import { DEFAULT_TAB } from '../consts';
+import { DEFAULT_TAB, TABS_SLOT_ID } from '../consts';
 import OverviewTab from './Overview';
 
 export const registerCoreTabs = () => {
   addGlobalFill(
-    'host-details-page-tabs',
+    TABS_SLOT_ID,
     DEFAULT_TAB,
     <OverviewTab key="host-details-overview-tab" />,
-    1000
+    1000,
+    { translations: { overview: __('Overview') } }
```

Comment:
I might be missing something, why not use?
```suggestion
    { title: __('Overview') }
```

Reply:
a slot mostly has multiple fills,  while the [selector](https://github.com/theforeman/foreman/pull/8695/files#diff-1d486cf5c655a2d35d078eb58bc7348ccc80d2fc262e6849f69914f22a9c7bfaR43) returns the entire slot metadata. using the same key ('title') across fills will override its value.